go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/pedroalbanese/jwt-cracker
```

Comment:
This needs to change to github.com/alexrsagen/jwt-cracker for it to work after merge.